modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/SectionSubscription.es.js

```diff
@@ -36,9 +37,8 @@ export default ({
 
 	const onCompleted = () => {
 		setSubscription(!subscription);
-		if (onSubscription) {
-			onSubscription(!subscription);
-		}
+		deleteCache();
+		onSubscription?.();
```

Comment:
there is a 4th option that is to keep the state of the subscription in the component and ignore the result of the query, that's why this callbacks usually propagate the status.